pkg/services/ngalert/models/instance.go

```diff
@@ -90,22 +90,22 @@ type ListAlertInstancesQueryResult struct {
 }
 
 type FetchUniqueOrgIdsQueryResult struct {
-	DefinitionOrgID int64 `xorm:"def_org_id" json:"definitionOrgId"`
+	DefinitionOrgID int64 `xorm:"rule_org_id" json:"definitionOrgId"`
```

Comment:
should this field and the json tag change as well?

Reply:
Oh forgot about this one, can change. I actually figured I will just change it to return a slice and simplify it. Will do that with this.